### DISCUSSION TITLE
Remote Student Application- Keep getting rejected

### DISCUSSION THREAD

**Discussion by josephk96** [community/community#46668] [GitHub Education] [answered]
My university is in the United States, and I'm in Korea. I'm a full time remote student, and I submitted an unofficial transcript screenshot showing that my campus is `Online`, but I keep getting rejected. I've opened a support ticket for this roughly two weeks ago, but I haven't received any replies. This is getting ridiculous - I understand that this GH is being generous with this Student Pack program; however, I can't understand why they keep on rejecting my applications, even when the proof they requested clearly indicates that I am an online student. Has anyone been rejected for the same reason? If so, how did you manage to convince GH that you are in fact, a student of the institution that you claim to be a part of?

Oh, what's even more frustrating is that I also have a valid email address from my institution, but instead of asking for additional verification, I just get an automated rejection message saying that the proof I submitted is not sufficient because it doesn't indicate that I'm enrolled in an online/remote program.

**Comment by aanchal-adk** [comment, marked as answer]
I'm facing the same issue. I live in Nepal and doing my online masters from Georgia Tech located in US. Please provide a resolution for this issue faced by us international online students.

**Reply by josephk96** [reply]
Yeah the response I got 2 days ago in the support ticket I created 25 days ago is ridiculous - 

> Hi there,
> 
> Thank you for reaching out to GitHub Support. This ticket has been closed.
> Please direct your inquiries for GitHub Education to our [GitHub Community](https://github.com/orgs/community/discussions/categories/github-education).
> 
> If your issue has not been resolved:
> 
> Check out our [GitHub Education documentation](https://docs.github.com/education) for answers to commonly asked questions Check the status of your application [here](https://education.github.com/discount_requests/pack_application)
> Search for similar questions and solutions in the [GitHub Community](https://github.com/orgs/community/discussions/categories/github-education)
> You can comment on an existing post if you have a similar issue.
> If your question has not been asked yet, post a question in the [GitHub Education Discussions](https://github.com/orgs/community/discussions/categories/github-education) including as much information as you can, but please do not include personally identifiable information like your email or mailing address.
> Helpful links to some of our more commonly asked GitHub Education questions:
> [Student Developer Pack Application & FAQs](https://github.com/orgs/community/discussions/17814)
> [Is Canva Pro still offered in the student account offers?](https://github.com/orgs/community/discussions/28136)
> 
> Ways to help other students:
> 
> Help answer questions: 🎒[GitHub Education in Community Discussions](https://github.com/orgs/community/discussions/categories/github-education) is a dedicated space for feedback, questions and conversations about anything GitHub Education related.
> Join the conversation and meet fellow students and teachers all learning together with GitHub. Share your experience, help improve the community, and continue to develop your GitHub skills!
> For additional guidance, please review our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service).
> 
> Thank you!
> GitHub Support

If GitHub really does not want to provide the Education Pack to remote students, they could just state that upfront. However, instead of doing so, they keep on rejecting my verification requests saying that there is insufficient proof that I am a remote student, even though I submitted a screenshot of my unofficial transcript for this current semester which shows that the campus is `Online`.

It's 2023 folks, learning can be done virtually. In fact, my institution has stated that they do not differentiate between their online students and in-campus student - even our diplomas are exactly the same. If GitHub has an issue with this, as I mentioned earlier they could either change their Education Pack ToS to reflect the fact that they are not cool with students being not physically located in the same city as their institution OR they could request additional information instead of just blindly rejecting all remote students' applications.

**Reply by aanchal-adk** [reply]
I finally got accepted! I had to use the college VPN and restart the application from the start. I was getting rejected even after using the VPN initially. After lots of attempts, github prompted me to restart the application. After resubmitting again, it worked. I had tried to cancel the application and restart from the beginning before but I wasn't able to get out of the document upload page for some reason. And after around 13 rejections, github cancelled the application altogether and I was able to start the application from the beginning. So my point is, try to start from the beginning of the application and use college vpn. For the document image, I used the bursar payment invoice. Hope it helps!
More importantly, hope github makes it easier for online remote students to apply.

**Reply by josephk96** [reply]
@aanchal-adk , I did this and finally got accepted! Really appreciate you sharing this and apologies for the late reply, GIOS has been keeping me very occupied haha

**Reply by eniolaseun1** [reply]
> I finally got accepted! I had to use the college VPN and restart the application from the start. I was getting rejected even after using the VPN initially. After lots of attempts, github prompted me to restart the application. After resubmitting again, it worked. I had tried to cancel the application and restart from the beginning before but I wasn't able to get out of the document upload page for some reason. And after around 13 rejections, github cancelled the application altogether and I was able to start the application from the beginning. So my point is, try to start from the beginning of the application and use college vpn. For the document image, I used the bursar payment invoice. Hope it helps! More importantly, hope github makes it easier for online remote students to apply.

What do you mean by school vpn?

**Reply by aanchal-adk** [reply]
@eniolaseun1 I'm terribly sorry for the late reply. I hope you've figured it out by now. Just in case, I meant the VPN service that schools/universities provide so that students can remotely access the school's resources.  The reason for using the VPN is that when you're browsing the internet, it will seem as if you're accessing it from your campus location. Github checks to see if the applicant is currently near the location of the campus.

**Comment by ChefAndy** [comment]
The lack of one hand talking to the other is just astonishing. My school has their entire operation on Microsoft hosted services and I've got an active student SSO account. No good? This is exactly the kind of thing people were worried about when MS acquired GH. 

My fairly large school won't issue school IDs for the new year until we show up on campus, so I can't do that. They don't have a VPN so I can't sign up through that. My transcript doesn't have my fall classes. My schedule would be some wonky elucian screenshot that I couldn't even get all on the same page at once. Being the only student with development experience in our senior game design studio next year, I need a solution up-and-running before class starts. Well, I guess that leaves me no choice but to set up a server and get my 12ish classmates acclimated with Perforce instead of git. Shortsighted.

**Comment by semhaydn** [comment]
I am having the same problem, created two tickets and asked a community question but unfortunately no support or help...
